clients/cloud/python/producer.py

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env python
+#
+# Copyright 2018 Confluent Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# =============================================================================
+#
+# Produce messages to Confluent Cloud
+# Using Confluent Python Client for Apache Kafka
+#
+# =============================================================================
+
+from confluent_kafka import Producer, KafkaError
+from confluent_kafka.admin import AdminClient, NewTopic
+import sys
+import json
+
+
+if __name__ == '__main__':
+    if len(sys.argv) != 3:
+        sys.stderr.write('Usage: %s <path to Confluent Cloud configuration file> <topic name>\n' % sys.argv[0])
+        sys.exit(1)
+
+    # Read Confluent Cloud configuration for librdkafka clients
+    config_file = sys.argv[1]
+    conf = {}
+    with open(config_file) as fh:
+      for line in fh:
+        line = line.strip()
+        if line[0] != "#" and len(line) != 0:
+          parameter, value = line.strip().split('=', 1)
+          conf[parameter] = value.strip()
```

Comment:
I wonder if this is worth printing out (in the consumer too).

Or if librdkafka prints it in a different debug level, maybe document how to set the debug level in Python.

Reply:
@gwenshap can you please elaborate, when you write "this is worth printing out" -- what is _this_?

In case you mean the `bootstrap.server`, `sasl.username`, and `sasl.password`, I actually deliberately didn't print that out to keep api key/password private and not share in stdout

If you mean something else, please let me know.

Reply:
I meant printing out the config (without the security bits) - the consumer group part can be important for instance. 

No strong feelings, but Java does log configs.

Reply:
My strong opinion is that the Java client shouldn't print anything unless explicitly told to, since it is a library, not an application.
In this case we're making an application, but I'd consider the default printing of config an anti-pattern, it doesn't serve much purpose and just gets in the way, possibly exposing secrets in (CI, et.al) output.

Reply:
Leaving this out